src/main/java/io/pravega/connectors/hadoop/PravegaInputFormatBuilder.java

```diff
@@ -56,19 +56,39 @@ public PravegaInputFormatBuilder withURI(String uri) {
         return builder();
     }
 
-    @Override
+    /** Adds Deserializer class name to builder.
+     *
+     * @param className String
+     * @return builder instance
+     */
     public PravegaInputFormatBuilder withDeserializer(String className) {
         this.setString(PravegaConfig.INPUT_DESERIALIZER, className);
         return builder();
     }
 
-    @Override
+    /**
+     * Adds optional start positions to builder
```

Comment:
Where is the application supposed to get the start position from? Could you be more specific? Is that from a stream cut? To me, it feels more natural that it is out of a stream cut, but if not, it needs to be discussed here.